client/src/components/pages/auth/Register.js

```diff
@@ -102,9 +103,9 @@ export class Register extends Component {
     render() {
         return (
             <div className="register-container">
-                <Link to="/" className="button small tertiary">
+                <GoHome to="/" className="button small tertiary" buttonOrLink="button" size="small" kind="tertiary">
```

Comment:
I take back what i said about this. I would revert this because you will never be able to register when not logged in. I don't know what I was thinking before. Sorry😬

Reply:
🐋